stl/inc/xstring

```diff
@@ -4600,6 +4616,31 @@ template <class _Elem, class _Traits, class _Alloc>
 _NODISCARD bool operator>=(const basic_string<_Elem, _Traits, _Alloc>& _Left, _In_z_ const _Elem* const _Right) {
     return !(_Left < _Right);
 }
+#endif // !_HAS_CXX20
+
+#if _HAS_CXX20
+template <class _Elem, class _Traits, class _Alloc>
+_NODISCARD strong_ordering operator<=>(

```

Comment:
The return type is incorrect. See [[string.cmp]](https://eel.is/c++draft/string.cmp#itemdecl:1) and [[string.view.comparison]](https://eel.is/c++draft/string.view.comparison#lib:operator%3c=%3e,basic_string_view).

Same below.